src/routes/(shell)/+page.svelte

```diff
@@ -29,8 +29,8 @@
 					</div>
 					<div class="flex items-center gap-4">
 						<ProgressRing
-							value={quota.used}
-							max={quota.quantity}
+							value={(quota.quantity <= 0) ? 0 : quota.used / quota.quantity * 100}
+							max=100
```

Comment:
Actually that's the default, so can be deleted.

Reply:
`100` is the default, but I wanted to be explicit about transforming the quota value into a percentage. (Although, if I'm set on that, I should also supply the format which defaults to percentage. Ehhh, OK, let's replace it with a comment.)